float32[6] handle_direction
sensor_msgs/PointCloud handle_pc
shape_msgs/Plane drawer_plane